api/types/accesslist/inclusion.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package accesslist
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/gravitational/trace"
+)
+
+// Inclusion values indicate how membership and ownership of an AccessList
+// should be applied.
+type Inclusion uint
```

Comment:
I wasn't foreseeing this much boilerplate. I suppose you could keep the Go enum as a `type string` and map to/from the protos to it, if that's simpler.